src/components/WorkOrderDetails.js

```diff
@@ -354,12 +355,16 @@ class WorkOrderDetail extends Component {
             <AccordionItemBody>
               <Button
                 icon={faCamera}
-                text={"New Image"}
+                text={"Take Picture"}
                 linkPath={`/work-order/add-image/${
                   this.props.match.params.workOrderId
                 }`}
               />
-              {this.state.imagesData.length === 0 && <p>No data</p>}
+              <UploadImage
```

Comment:
Could we put these buttons side by side instead of stacked? Will save us a small amount of vertical screen real-estate and might also prevent any fat thumb issues.

I think this would do the trick:
```html
  <div class="row">
    <div class="col">
      ... button
    </div>
    <div class="col">
      ... button
    </div>
  </div>
```

Reply:
For sure! 👍 Made this change and it looks a lot better.

![Screen Shot 2019-10-30 at 11 01 51 AM](https://user-images.githubusercontent.com/37249039/67875689-b8c6fb80-fb04-11e9-865c-83a85bb95cd6.png)